meson_options.txt

```diff
@@ -65,10 +65,10 @@ option(
     description: 'Enable build of PGP UAM module',
 )
 option(
-    'enable-quota',
-    type: 'feature',
-    value: 'auto',
-    description: 'Turn on quota support',
+    'with-quota',
+    type: 'string',
+    value: '',
+    description: 'Turn on quota support (default: true)',
```

Comment:
How will users know that they can pass a path string to this option?

Reply:
Good point. Will revert this change until we deal with option semantics as I need this one to be merged first

Reply:
SGTM